### DISCUSSION TITLE
How to customize Slack Alert?

### DISCUSSION THREAD

**Discussion by marcosmas28** [semaphoreui/semaphore#2149] [Q&A]
Hi,
I'm new to Semaphore, I've been testing its features and I really like this app. It has everything I need to run my ansible playbooks: log viewer, scheduler, github integration, and slack alerts.
Now I would like to customize the message I send from Semaphore to Slack.
What I have in mind is to send the final "PLAY RECAP" from the ansible-playbook execution.
How could I do this? Is it configurable? or should I tweak Semaphore code?
Any idea to accomplish this?
Thank in advance!

**Comment by jaredt87** [comment]
I'm a fairly heavy user of semaphore, have automated my entire monthly patching, and have nightly processes that run. I'd like this feature too so that any end users who want to see what things are occurring can just go to slack and see what happened.

**Comment by Random6554** [comment]
Any update on this feature?

**Comment by mattboston** [comment]
Following. As I mentioned in one of the feature request issues, I would love to have the ability to customize alerts using a template.
